### ISSUE TITLE
in python, loos.fileopenerror prints unicode garbage instead of file-path.

### ISSUE THREAD

**Issue by lgsmith** [GrossfieldLab/loos#79]
**Describe the bug**
When loos can't open a file, it emits an error. For the last several months, it has emitted errors with unicode that don't print properly to terminal.

**To Reproduce**
- open a python interpreter that can import loos as installed by the `conda_build.sh` script
- say `loos.createSystem('foo.pdb')` as long as that file isn't in your cwd.
You'll see:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/louis/miniconda3/envs/loos-enspara/lib/python3.10/site-packages/loos/__init__.py", line 3881, in createSystem
    return _loos.createSystem(*args)
loos.FileOpenError: @\udcccM\udcd9\udccbU
```

**Expected behavior**
I'd expect it to tell me the name of the file it couldn't open

**LOOS version and platform
Which version you're running, on what OS, and how it was built (e.g. conda, system libraries, manually installed dependencies)

conda_build.sh, up-to-date main. Note: older builds do not do this: I think someone changed the error handling, because on older builds the error looks like this:
```
>>> loos.createSystem('foo.pdb')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/louis/miniconda3/envs/loos/lib/python3.9/site-packages/loos/__init__.py", line 3881, in createSystem
    return _loos.createSystem(*args)
loos.FileOpenError
```

**Additional context**
Aside from being annoying that I don't get told which file couldn't be read, this also creates confusion. I thought I'd somehow mis-compiled something, and that's why I was having this problem; it took some time testing things to feel confident that I wasn't having other issues, and that this was just some kind of bug. When some labmates of mine encountered this, they had a similar reaction, so I think it's bad. If the real filename can't be provided cleanly, probably just not reporting it as was done previously is a less confusing response than printing improperly displayed unicode.

**Comment by lgsmith** [comment]
Did we ever get a PR that resolves this?

**Comment by agrossfield** [comment]
No, but it’s on the todo list for September, if Tod is able to make time after the release.

**Comment by agrossfield** [comment]
This is fixed in LOOS 4.0
